webpack.config.js

```diff
@@ -8,7 +8,7 @@ module.exports = /** @type {import('webpack').Configuration} */ ({
     contact_us_form_element: './assets/js/contact_us_form_element.ts',
     faqs: './assets/js/faqs.js',
     partners_contact: './assets/js/partners/contact.js',
-    touchpoints_translations: './assets/js/touchpoints_translations.js',
+    touchpoints: ['./assets/js/touchpoints_translations.js', './assets/js/touchpoints_58f51d4d.js'],
```

Comment:
does order matter here? like would we want to load touchpoints first *and then* load translations for it?

Reply:
makes sense!